specifications/staticfile.yaml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.0
+servers: []
+info:
+  version: 0.1.0
+  title: Static file yaml
+  description: This is a API specification for the static files of the mock server
+  contact:
+    name: MaaS Alliance
+    url: 'https://maas-alliance.eu/'
+  license:
+    name: MIT
```

Comment:
I believe the APIs here are licensed under Apache 2.0

Reply:
changed it to Apache 2.0